pgs_point.sql.in

```diff
@@ -28,6 +34,9 @@ CREATE FUNCTION set_sphere_output(CSTRING)
 COMMENT ON FUNCTION spoint(FLOAT8, FLOAT8) IS
   'returns a spherical point from longitude (arg1), latitude (arg2)';
 
+COMMENT ON FUNCTION spoint_deg(FLOAT8, FLOAT8) IS
+  'returns a spherical point from longitude(in degrees) (arg1), latitude(in degrees) (arg2)';
```

Comment:
Change to `returns a spherical point from longitude (arg1, in degrees), latitude (arg2, in degrees)`.